### DISCUSSION TITLE
Problem when starting

### DISCUSSION THREAD

**Discussion by TheBrettYT** [eritislami/evobot#434] [Q&A]
When i try starting the bot its says i need Discord.Js
Does node_modules have to be in there for the discord bot to work???
Please fix this problem mine just failed today

**Comment by evrardco** [comment]
This is not a bug. For things such as asking help for building the project, you'd be better off doing that in the discussions tab of the project, I'll be happy to help you there :)

**Comment by eritislami** [comment]
Please read instructions before creating issues, it says that you have to run `npm install` after cloning

**Comment by chiquidev** [comment]
Hi! Good night, for correct this issue you should type at the console `npm i discord.js`
